classes/install/Upgrade.inc.php

```diff
@@ -549,6 +549,34 @@ private function _transferSignoffData($signoffId, $queryId) {
 
 		$submissionFileDao->update('DELETE FROM signoffs WHERE signoff_id=?', array($signoffId));
 	}
+
+	/**
+	 * If StaticPages table exists we should port the data as NMIs
+	 * @return boolean
+	 */
+	function migrateStaticPagesToNavigationMenuItems() {
+			if ($this->tableExists('static_pages')) {
```

Comment:
Looks like this is indented with double tabs -- use single tabs.

Reply:
fixed